README.md

```diff
@@ -38,7 +40,9 @@ psvm -v "1.7.0" -o
 # List all available Polkadot SDK versions
 psvm -l
 # Check against a particular Polkadot SDK version without updating the Cargo.toml file
-psmv -v "1.4.0" -c
+psvm -v "1.4.0" -c
+# Update the ORML dependencies along with the Polkadot SDK dependencies.
+psvm -v "1.6.0" -O
```

Comment:
Makes me wonder if the usability is so great of this.

Maybe `psvm --sdk=1.6.0 --orml=1.5.0` would make more sense from a user perspective?

Reply:
Resolved in the issue.